packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -120,12 +120,12 @@ class UnThemedColorPickerPopover<T extends CustomPickersDescriptor> extends Reac
       <FocusScope contain restoreFocus autoFocus>
         <div className={styles.colorPickerPopover}>
           <div className={styles.colorPickerPopoverTabs}>
-            <div tabIndex={0} className={this.getTabClassName('palette')} onClick={this.onTabChange('palette')}>
+            <button className={this.getTabClassName('palette')} onClick={this.onTabChange('palette')}>
               Colors
-            </div>
-            <div tabIndex={0} className={this.getTabClassName('spectrum')} onClick={this.onTabChange('spectrum')}>
+            </button>
+            <button className={this.getTabClassName('spectrum')} onClick={this.onTabChange('spectrum')}>
               Custom
-            </div>
+            </button>
```

Comment:
if you're gonna change these, think you also need to change `renderCustomPickerTabs` function to return a `<button>` as well

Reply:
damn...already auto-merged @ashharrison90  😅